cryptofeed/standards.py

```diff
@@ -115,6 +115,7 @@ def timestamp_normalize(exchange, ts):
     },
     TRADES: {
         DYDX: 'v3_trades',
+        HITBTC: 'subscribeTrades',
```

Comment:
Why am I seeing this here? It supported trades before, are you adding a new one and leaving the old entry?